### ISSUE TITLE
[Device Support Request] Nedis ZBHTR20WT TS0601 _TZE200_ne4pikwm Smart Thermostatic Radiator

### ISSUE THREAD

**Issue by bpcrocke** [zigpy/zha-device-handlers#3517]
### Problem description

The Device is recognized but does not show any settings. 

### Solution description

Please add to database. 

### Screenshots/Video

<details><summary>Screenshots/Video</summary>

[Paste/upload your media here]

</details>

### Device signature

<details><summary>Device signature</summary>

```json
{
  "node_descriptor": {
    "logical_type": 2,
    "complex_descriptor_available": 0,
    "user_descriptor_available": 0,
    "reserved": 0,
    "aps_flags": 0,
    "frequency_band": 8,
    "mac_capability_flags": 128,
    "manufacturer_code": 4417,
    "maximum_buffer_size": 66,
    "maximum_incoming_transfer_size": 66,
    "server_mask": 10752,
    "maximum_outgoing_transfer_size": 66,
    "descriptor_capability_field": 0
  },
  "endpoints": {
    "1": {
      "profile_id": "0x0104",
      "device_type": "0x0051",
      "input_clusters": [
        "0x0000",
        "0x0004",
        "0x0005",
        "0xef00"
      ],
      "output_clusters": [
        "0x000a",
        "0x0019"
      ]
    }
  },
  "manufacturer": "_TZE200_ne4pikwm",
  "model": "TS0601",
  "class": "zigpy.device.Device"
}
```

</details>

### Diagnostic information

<details><summary>Diagnostic information</summary>

```json
[Paste the diagnostic information here]
```

</details>

### Logs

<details><summary>Logs</summary>

```python
[Paste the logs here]
```

</details>

### Custom quirk

<details><summary>Custom quirk</summary>

```python
[Paste your custom quirk here]
```

</details>

### Additional information

More info about this device can be found over at zigbee2mqtt: https://github.com/Koenkk/zigbee2mqtt/issues/21202

Here are the device codes: https://github.com/Koenkk/zigbee2mqtt/files/14591996/ZBTHR20WT.Codes.pdf

The manufacturer's website can be found here: https://nedis.de/de-de/product/haushalt-und-wohnen/klima/heizung/550786152/smartlife-heizkorpersteuerung-zigbee-30-batteriebetrieben-led-android-ios

**Comment by Hely0n** [comment]
I'd appreciate it too. Ordered one, but now it has to wait on the shelf 😁.

@bpcrocke, how did you get it into pairing mode? I coudn't find any instruction about it.

**Comment by bpcrocke** [comment]
Just follow the manual: https://cdn.nedis.com/manuals/html/ZBHTR20WT/?locale=en-GB

**Comment by Hely0n** [comment]
Ah, now I understand. They mean you have to turn it down until "off"-state and only after that the pairing mode works by holding 5sek. Thank you

**Comment by sonikki789** [comment]
> I'd appreciate it too. Ordered one, but now it has to wait on the shelf 😁.

Same here. Bought one and thought pairing this would be a breeze 😁 I´ve been fiddling around with several quirks for similar thermostats, but haven´t had any luck, so huge respect if someone has time and effort to put on this.

**Comment by wsrtdc** [comment]
Me too, I would appreciate if someone could make work this device under zha - Many thanks in advance!

BTW: ZBHTR10WT is working (says blakadder), so it's a real pity, that ZBHTR20WT don't work!

**Comment by BerT666** [comment]
Same issue here...
Could pair the device, but do not get any data :-(


TS0601
by _TZE200_ne4pikwm

I managed to get "some things" working with the quirk "ts0601_trv.Thermostat_TZE200_c88teujp"...
At least I can set Temp / turn on or off and get the temp of the trv ;-)

Hope this helps to get a "fitting" quirk faster ;-)

**Comment by checkiecheck** [comment]
that pull request, is it possible to test it as custom quirk? (or any idea when this would be merged into HA release?)
